cmd/server-manager/views/pages/results/index.html

```diff
@@ -5,7 +5,7 @@
 {{ define "content" }}
     <div class="row">
         <div class="col-md-4">
-
+            <a href="/results/combine" class="btn btn-primary">Combine Results</a>
```

Comment:
this needs to be inside an if write access. As does the upload one below, which i'm kinda annoyed we didn't see before.